### PR TITLE
Moderation - The new

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Fixes my horrible un-moderation of PRs in the past

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
... What the fuck Triv

**Comment by kevinz000** [comment]
needs to be added to .dme

**Comment by nik707** [comment]
This can't be merged without being added to the .dme, which has to be done by compiling the code

**Comment by nik707** [comment]
trilby no. they need to be .dm files but the files have to be in the vorestation.dme which can't be done without compiling the code

File: modular_citadel/code/modules/mob/living/silicon/robot/robot_modules/station.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/obj/item/weapon/robot_module/drone/New(var/mob/living/silicon/robot/robot)
```
Comment (kevinz000) [inline modular_citadel/code/modules/mob/living/silicon/robot/robot_modules/station.dm]: no var/

**Comment by kevinz000** [comment]
also combine the two prs

File: vorestation.dme [hunk 2]
```diff
@@ -1,3068 +1,3073 @@
-
```
[the remark this answers is not in the document]
Comment (Trilbyspaceclone) [reply vorestation.dme]: I dont know what the heck your asking me to do

**Pull request closed without merging**